python-threatexchange/threatexchange/content_type/file_content.py

```diff
@@ -0,0 +1,30 @@
+import typing as t
+from content_base import ContentType
+from photo import PhotoContent
+from video import VideoContent
+from PIL import Image
+
+class FileContent(ContentType):
+    """
+    ContentType representing a generic file. 
+    
+    Determines if a file is a photo or video based on file extension.
+    """
+
+    VALID_PHOTO_EXTENSIONS = {ext.lower() for ext in Image.registered_extensions()}
```

Comment:
This is doing work during module import time, which is generally considered an anti-pattern, but I don't think it's worth changing

Reply:
Could you provide more details?

Reply:
Could you tell me more details?

Reply:
Here's the first result for me from google for "doing work during module import time"
https://www.benkuhn.net/importtime/

As mentioned, I don't think it's worth changing.